src/Dockerfile-minimal.template

```diff
@@ -49,7 +49,7 @@ LABEL summary="$SUMMARY" \
       io.openshift.expose-services="8080:http" \
       io.openshift.tags="builder,python,python{{ spec.short_ver }},python-{{ spec.short_ver }},rh-python{{ spec.short_ver }}" \
       com.redhat.component="python-{{ spec.short_ver }}-container" \
-      name="{{ spec.org }}/python-{{ spec.short_ver }}-minimal" \
+      name="{{ spec.org }}/python-{{ spec.short_ver }}-{% if spec.org == "sclorg" %}{{ spec.prod }}-{% endif %}minimal" \
```

Comment:
Ad `python-39-c9s-minimal`: Here's where it should be changed if we decide to do so.